setup.py

```diff
@@ -41,6 +41,7 @@
         "neural-compressor>=2.2.0",
         "onnx",
         "onnxruntime<1.15.0",
+        "intel-extension-for-transformers",
```

Comment:
I don't think we need to make `intel-extension-for-transformers` a required dependency, why not make it optional ? (like we did for nncf and openvino for example : https://github.com/huggingface/optimum-intel/blob/65b20f5f71f2619c07231221adae295d1b117bbc/optimum/exporters/openvino/__main__.py#L126)